test/test_other.py

```diff
@@ -15265,26 +15265,34 @@ def test_preload_module(self, args):
       #include <emscripten.h>
       #include <emscripten/threading.h>
 
+      #ifdef __EMSCRIPTEN_PTHREADS__
+      EM_JS_DEPS(main, "$sharedModules");
+      #endif
+
       int main() {
         // Check the file exists in the VFS
         struct stat statbuf;
         assert(stat("/library.so", &statbuf) == 0);
 
         // Check that it was preloaded.
         // The preloading actually only happens on the main thread where the filesystem
-        // lives.  On worker threads the module object is shared via preloadedModules.
+        // lives.  On worker threads the module object is shared via sharedModules.
         if (emscripten_is_main_runtime_thread()) {
           int found = EM_ASM_INT(
             return preloadedWasm['/library.so'] !== undefined;
           );
           assert(found);
-        } else {
+        }
+      #ifdef __EMSCRIPTEN_PTHREADS__
+        else {
           int found = EM_ASM_INT(
             err(sharedModules);
             return sharedModules['/library.so'] !== undefined;
           );
           assert(found);
         }
+      #endif
```

Comment:
Was this change really needed?  I think if this code worked prior to this change it should work after it no?

i.e. the `sharedModules` symbol is available in the same builds as before, right?

Reply:
Seems to be unnecessary, reverted with commit da0b4df48a9bc1f6d55909416fdc71e90591ff44.